docs/man_pages/project/testing/publish.md

```diff
@@ -0,0 +1,44 @@
+publish
```

Comment:
Is this like an alias of appstore upload? Or you have forgotten to remove this command?